tests/grf/test_core.py

```diff
@@ -0,0 +1,95 @@
+import unittest.mock
+
+import numpy as np
+import pytest
+
+import glass.grf
```

Comment:
Again, we should stick to one convention imo. We can switch to this convention in a separate PR, but for this PR I think it would be better to stick to the existing one.

Reply:
I really don't like `from glass import ...`, also for users. Happy to have a separate PR where we switch everything over to `import glass`.